graylog2-web-interface/src/components/search/MessageFields.jsx

```diff
@@ -19,7 +19,7 @@ const MessageFields = React.createClass({
 
   _formatFields(fields, showDecoration) {
     if (!showDecoration || !this.props.message.decoration_stats) {
-      return Object.keys(fields).sort().map(key => <MessageField {...this.props} fieldName={key} value={fields[key]} />);
+      return Object.keys(fields).sort().map(key => <MessageField key={key} {...this.props} fieldName={key} value={String(fields[key])} />);
```

Comment:
Do we really want to convert it into a string? Wouldn't it make more sense to relax the prop type for the component? It might make sense to still have the original type in the component, so we could display numbers/whatever differently.